compute/disk-images/disk_image_test.go

```diff
@@ -81,6 +81,11 @@ func TestComputeDiskImageSnippets(t *testing.T) {
 	imageName := fmt.Sprintf("test-image-go-%v-%v", time.Now().Format("01-02-2006"), r.Int())
 	diskName := fmt.Sprintf("test-disk-go-%v-%v", time.Now().Format("01-02-2006"), r.Int())
 	sourceImage := "projects/debian-cloud/global/images/family/debian-11"
+	sourceProjectId := "debian-cloud"
+	sourceImageName := "debian-11"
+	guestOsFeatures := []*computepb.GuestOsFeature{
```

Comment:
this is a bit confusing, as you're creating a debian image, and labeling it as a windows image? is there a reason we need to include Guest OS Features here, or can they be left out to simplify the sample?

Reply:
Yeah, makes sense to simplify. I was looking into the documentation and wanted to show the usage of the parameter... I agree it makes the function more general purpose, so I removed it.